test/Npgsql.CrateDb.Tests/CrateDbIntegrationTest.cs

```diff
@@ -0,0 +1,174 @@
+﻿using System;
+
+namespace Npgsql.CrateDb.Tests
+{
+    class CrateIntegrationTest : TestBase
+    {
+        protected void CreateTestTable()
+        {
+            using (var con = OpenConnection())
+            using (var cmd = con.CreateCommand())
+            {
+                cmd.CommandText = @"create table if not exists test(
+                    id integer primary key,
+                    string_field string,
+                    boolean_field boolean,
+                    byte_field byte,
+                    short_field short,
+                    integer_field integer,
+                    long_field long,
+                    float_field float,
+                    double_field double,
+                    timestamp_field timestamp,
+                    object_field object(strict) as (""inner"" string),
+                    ip_field ip,
+                    geo_point_field geo_point,
+                    geo_shape_field geo_shape
+                    ) clustered by (id) into 1 shards with (number_of_replicas=0)";
```

Comment:
Same as the comment :point_down: 

```suggestion
                    ) clustered by (id) into 1 shards with (number_of_replicas=0, column_policy = 'dynamic')";
```